cypress/integration/channel/delete_parent_message_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getRandomInt} from '../../utils';
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 4] */
+
+describe('Delete Parent Message', () => {
+    before(() => {
+        // 1. Go to Main Channel View with "user-1"
+        cy.toMainChannelView('user-1');
+    });
+
+    it('C14270 Deleting parent message should also delete replies from center and RHS', () => {
+        // 2. Post a message and add replies
+        cy.postMessage('Parent Message');
+
+        cy.getLastPostId().then((postId) => {
+            cy.clickPostCommentIcon(postId);
+
+            // * Check that the RHS is open
+            cy.get('#rhsContainer').should('be.visible');
+
+            // * Add replies (randomly between 1 to 3)
+            const replyCount = getRandomInt(2) + 1;
+            for (var i = 0; i < replyCount; i++) {
+                cy.get('#reply_textbox').type('Reply').type('{enter}');
+
+                // add wait time to ensure that a post gets posted and not on pending state
+                cy.wait(500); // eslint-disable-line
+            }
+
+            cy.getLastPostId().then((replyPostId) => {
+                // * No delete modal should be visible yet
+                cy.get('.modal-body').should('not.be.visible');
```

Comment:
Please add element ID so we can access it with such new ID instead of CSS class selector.  Same applies to `.modal-footer` below.

Reply:
Reached out to @letsila, will be updating my PR after his changes (adding ids to delete_post_modal component) go in.

Good thinking on finding a use for this double work by the way!